plugins/aea-crypto-cosmos/setup.py

```diff
@@ -36,4 +36,22 @@
         "aea.ledger_apis": ["cosmos = aea_crypto_cosmos:CosmosApi"],
         "aea.faucet_apis": ["cosmos = aea_crypto_cosmos:CosmosFaucetApi"],
     },
+    classifiers=[
+        "Environment :: Console",
+        "Environment :: Web Environment",
+        "Development Status :: 2 - Pre-Alpha",
```

Comment:
We should change this status when we do v1. Let's open an issue so we don't forget.